libs/components/layout/src/lib/modules/action-button/action-button-container.component.ts

```diff
@@ -44,54 +44,66 @@ export class SkyActionButtonContainerComponent
    */
   @Input()
   public set alignItems(value: SkyActionButtonContainerAlignItemsType) {
-    this._alignItems = value;
+    this.#_alignItems = value;
   }
 
   public get alignItems(): SkyActionButtonContainerAlignItemsType {
-    return this._alignItems || 'center';
+    return this.#_alignItems || 'center';
   }
 
   @ContentChildren(SkyActionButtonComponent)
-  private actionButtons: QueryList<SkyActionButtonComponent>;
+  private actionButtons: QueryList<SkyActionButtonComponent> | undefined;
 
   @ViewChild('container', {
     read: ElementRef,
     static: true,
   })
-  private containerRef: ElementRef<any>;
+  private containerRef: ElementRef | undefined;
 
-  private ngUnsubscribe = new Subject<void>();
+  #ngUnsubscribe = new Subject<void>();
 
-  private syncMaxHeightTimeout?: number;
+  #syncMaxHeightTimeout?: number;
 
   private set themeName(value: string) {
-    this._themeName = value;
+    this.#_themeName = value;
     this.updateResponsiveClass();
     this.updateHeight();
   }
 
-  private _alignItems: SkyActionButtonContainerAlignItemsType;
+  #_alignItems: SkyActionButtonContainerAlignItemsType | undefined;
 
-  private _themeName: string;
+  #_themeName: string | undefined;
```

Comment:
I wonder if this should default to `default`?

Reply:
I don't disagree, but I think that change makes more sense if we had a more strongly typed theme name